### ISSUE TITLE
Enhancement - Show Tough value along with Quality/Defense

### ISSUE THREAD

**Issue by Majawat** [erkannt/text2tabletop#4]
It'd be nice to see the tough value as a larger number up top near the Quality and Defense numbers

**Comment by Maltavius** [comment]
Yes! With a heart next to it, or even checkboxes so that you can mark them of on the paper.
